server.go

```diff
@@ -220,6 +221,53 @@ type peerSlotStatus struct {
 	state peerAccessStatus
 }
 
```

Comment:
Maybe consider moving to a dedicated file like contractcourt/aux_closer_adapter.go for better organization. `server.go` is so long already.

Reply:
By extracting the types above into their own pkg the interface got simpler and the adapter was not needed after all.